### PR TITLE
Update lesson.md

### PR DESCRIPTION
**Pull request by RC-PM**
- Added numbers to section 
- Changed git to trello for backlog management 
- Highlighted facilitators for specific sessions

## What does this change?

Module:
Week(s):

### CODE REVIEW

File: docs/finalproject/week-1/lesson.md [hunk 1]
```diff
@@ -51,27 +51,34 @@ Once you have your team and your assignment, start by sorting out the basic admi
...
-### Project Management Boards (5 minutes)
-Project Management boards are a tool that you can use to help you organise how you work.
+### 2.3. Project Management Boards (5 minutes)
+*Led by students* 
+
+Project Management boards are a tool that you can use to help you organise how you work. 
 
 ```txt
-Create a [project board](https://help.github.com/en/articles/about-project-boards) in the repo for your tasks and stories
+Create a copy of this Project Board to manage your work - [Project Board](https://trello.com/b/wkQd3R5r/team-name-backlog)
```
Comment (textbook) [inline docs/finalproject/week-1/lesson.md:75]: Links won't work in code blocks (fences with backticks) - you can't use markdown syntax there

**Review by textbook — APPROVED** [review]
Looks good, unless you want to move that link

File: docs/finalproject/week-1/lesson.md (hunk 1 above)
Comment (RC-PM) [reply docs/finalproject/week-1/lesson.md:75]: Happy to change that - should I open another PR?

Comment (textbook) [reply docs/finalproject/week-1/lesson.md:75]: I'd add it to this one, it's all part of the same thing

**Pull request merged**